### PR TITLE
Adding object detection notebook

### PR DESCRIPTION
**Pull request by xzhou33**

### CODE REVIEW

**Comment by djarpin** [comment]
I get an error when opening these notebooks:
```
Unreadable Notebook: /home/ec2-user/SageMaker/efs/algos_frameworks_2018-07-11_launch_notebooks/Final/objdet/amazon-sagemaker-examples/object_detection_pascalvoc_coco/object_detection_image_json_format.ipynb NotJSONError('Notebook does not appear to be JSON: \'{\\n "cells": [\\n {\\n "cell_type": "m...',)
```

A bit difficult to track down where the malformed JSON might be.  Did you make a manual text edit somewhere that might have caused this?

**Comment by xzhou33** [comment]
Moved the object detection into the introduction_to_amazon_algorithms directory. Fixed the notebook json error.

File: introduction_to_amazon_algorithms/object_detection_pascalvoc_coco/object_detection_image_json_format.ipynb [hunk 1]
```diff
@@ -0,0 +1,677 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bucket = 'object-detection-ssd' # custom bucket name.\n",
+    "# bucket = sess.default_bucket()\n",
```
Comment (djarpin) [inline introduction_to_amazon_algorithms/object_detection_pascalvoc_coco/object_detection_image_json_format.ipynb]: Remove the custom bucket and uncomment this one.

File: introduction_to_amazon_algorithms/object_detection_pascalvoc_coco/object_detection_recordio_format.ipynb [hunk 2]
```diff
@@ -0,0 +1,526 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bucket = 'object-detection-ssd' # custom bucket name.\n",
+    "# bucket = sess.default_bucket() \n",
```
Comment (djarpin) [inline introduction_to_amazon_algorithms/object_detection_pascalvoc_coco/object_detection_recordio_format.ipynb]: Remove the above bucket and uncomment this line.

**Comment by xzhou33** [comment]
Removed the testing bucket name.

**Pull request merged**